src/aurelia-loader-webpack.ts

```diff
@@ -188,21 +195,23 @@ export class WebpackLoader extends Loader {
   * @param moduleId The module ID to load.
   * @return A Promise for the loaded module.
   */
-  async loadModule(moduleId: string, defaultHMR = true) {
+  loadModule(moduleId: string, defaultHMR = true) {
     let existing = this.moduleRegistry[moduleId];
     if (existing) {
-      return existing;
+      return Promise.resolve(existing);
     }
     let beingLoaded = this.modulesBeingLoaded.get(moduleId);
     if (beingLoaded) {
       return beingLoaded;
     }
     beingLoaded = this._import(moduleId, defaultHMR);
     this.modulesBeingLoaded.set(moduleId, beingLoaded);
-    const moduleExports = await beingLoaded;
-    this.moduleRegistry[moduleId] = ensureOriginOnExports(moduleExports, moduleId);
-    this.modulesBeingLoaded.delete(moduleId);
-    return moduleExports;
+    // const moduleExports = await beingLoaded;
+    return beingLoaded.then((moduleExports) => {
```

Comment:
Just saying: when you've got a single argument, braces are optional (I find it more legible without).